src/components/suggest/suggest.tsx

```diff
@@ -234,6 +234,11 @@ export const EuiSuggest: FunctionComponent<EuiSuggestProps> = ({
     ['State: loading.', 'State: saved.', 'State: unsaved.', 'State: unchanged.']
   );
 
+  const screenReaderInstructions = useEuiI18n(
+    'euiSuggest.screenReaderInstructions',
+    'Use Up and Down arrow keys to move focus over options. Press Enter to select. Press Escape to collapse options.'
```

Comment:
See above re: minor grammar suggestion
```suggestion
  const screenReaderInstructions = useEuiI18n(
    'euiSuggest.screenReaderInstructions',
    'Use the Up and Down arrow keys to move focus over options. Press Enter to select. Press Escape to collapse options.'
```